INSTALL.md

```diff
@@ -96,12 +96,13 @@ There are at least two ways to install git-ftp on Windows.
 ### Git for Windows, former msysgit (recommended)
 
 Install [Git for Windows](https://git-for-windows.github.io/).
-It comes with curl installed, but it doesn't support SFTP by default.
+~~It comes with curl installed, but it doesn't support SFTP by default.
 In order to use SFTP, [download curl](http://curl.haxx.se/download.html) for
 Windows with SFTP support.
 Win32 2000/XP MSI or Win64 2000/XP x86_64 MSI is recommended.
 If you installed curl, then remove `bin/curl.exe` from your Git for Windows
-installation directory. It will fall back to the newly installed version.
+installation directory. It will fall back to the newly installed version.~~
+Windows 10 comes now preinstalled with the correct cURL version.
```

Comment:
Can you specify which version of Windows 10 introduces the updated cURL version?
Again maybe a user has a older version. Then this information could be useful.
Maybe like:
```diff
+ Since Windows 10 version XXXX a cURL installation supporting SFTP is preinstalled.
```
And then the suggestion from above.

Reply:
If we do `curl -V` in Windows 10 2004 we have this:
> curl 7.55.1 (Windows) libcurl/7.55.1 WinSSL
> Release-Date: 2017-11-14, security patched: 2019-11-05
> Protocols: dict file ftp ftps http https imap imaps pop3 pop3s smtp smtps telnet tftp
> Features: AsynchDNS IPv6 Largefile SSPI Kerberos SPNEGO NTLM SSL

If we do `curl -V` in the latest Git Bash we have this:
> curl 7.71.1 (x86_64-w64-mingw32) libcurl/7.71.1 OpenSSL/1.1.1g (Schannel) zlib/1.2.11 brotli/1.0.7 libidn2/2.3.0 libssh2/1.9.0 nghttp2/1.41.0
> Release-Date: 2020-07-01
> Protocols: dict file ftp ftps gopher http https imap imaps ldap ldaps pop3 pop3s rtsp scp sftp smtp smtps telnet tftp
> Features: AsynchDNS brotli HTTP2 HTTPS-proxy IDN IPv6 Kerberos Largefile libz Metalink MultiSSL NTLM SPNEGO SSL SSPI TLS-SRP

So, AFAIK, there's no support for SFTP unless we use the cURL provided on the cURL website or the latest Git for Windows.
In fact it was only broken between version 2.16.2 and 2.25 versions. From version 2.25.1, SFTP support in the included cURL was restored.